apps/aggregates/src/utils/index.ts

```diff
@@ -35,18 +39,28 @@ export function sortData(
 
   if (dataType[0] === "number") {
     if (descending) {
-      return data.toSorted((current, next) => current[by] + next[by]);
+      return data.toSorted(
+        (current: Record<string, any>, next: Record<string, any>) =>
+          current[by] + next[by]
+      );
     }
-    return data.toSorted((current, next) => current[by] - next[by]);
+    return data.toSorted(
+      (current: Record<string, any>, next: Record<string, any>) =>
+        current[by] - next[by]
+    );
   }
 
   if (dataType[0] === "string") {
     if (descending) {
-      return data.toSorted((current, next) =>
-        current[by] < next[by] ? -1 : 1
+      return data.toSorted(
+        (current: Record<string, any>, next: Record<string, any>) =>
+          current[by] < next[by] ? -1 : 1
       );
     }
-    return data.toSorted((current, next) => (current[by] < next[by] ? 1 : -1));
+    return data.toSorted(
```

Comment:
Just a style preference, but I'd like to have explicit elses if the first if returns. I think that makes the code a bit more readable.

Reply:
That's also a good idea, but I might save it for the next version. Eventually, I'm going to rewrite (or get rid of it entirely) this function when we migrate the visualisation components over to tailwind. I need to expand the emx-chart components so that these data handling steps are handled by graphql.